Lib/test/test_funcattrs.py

```diff
@@ -93,6 +93,20 @@ def f(): print(a)
             self.fail("shouldn't be able to read an empty cell")
         a = 12
 
+    def test_set_cell(self):
+        a = 12
+        def f(): print(a)
+        c = f.__closure__
+        c[0].cell_contents = 9
+        self.assertEqual(c[0].cell_contents, 9)
```

Comment:
It would be nice to check how setting and deleting `cell_contents` affects the local variable `a`.